app/javascript/viewer/uv_manager.js

```diff
@@ -36,6 +37,31 @@ export default class UVManager {
     this.cdlTimer.initializeTimer()
   }
 
+  addIIIFIcon () {
+    const existingButton = document.querySelector('a.iiif-drag')
+    if (existingButton !== null) {
+      return
+    }
+    const shareButton = document.querySelector('button.share')
+    const link = document.querySelector('a.imageBtn.iiif').href
```

Comment:
I didn't know that was there! A comment describing this location would be helpful, something like `// pull the link from the iiif button inside the share popup`